src/callbacks/onResize/setPointSize.js

```diff
@@ -0,0 +1,20 @@
+export function setPointSize() {
+    var chart = this;
+    var config = this.config;
+    var points = this.svg.selectAll('g.point').select('circle');
+    if (config.point_size != 'Uniform') {
+        //create the scale
+        var sizeScale = d3.scale
+            .linear()
+            .range([2, 10])
+            .domain(d3.extent(chart.raw_data.map(m => m[config.point_size])));
+
+        //draw a legend (coming later?)
+
+        //set the point radius
+        points.transition().attr('r', function(d) {
```

Comment:
This `transition()` is overwriting the webcharts' transition of the points' coordinates, because the points never move when _Point Size_ is set to _ALP_.  Manually updating the `cx` and `cy` attributes seems to do the trick:
```
            .attr('cx', d => this.x(d.values.x))
            .attr('cy', d => this.y(d.values.y))
```

Reply:
good catch!